pkgs/by-name/im/immich-go/package.nix

```diff
@@ -48,6 +48,8 @@ buildGoModule rec {
     ldflags+=" -X github.com/simulot/immich-go/Date=$(cat SOURCE_DATE)"
   '';
 
+  __darwinAllowLocalNetworking = true;
```

Comment:
In general I would prefer to disable the affected tests, but not sure if this is possible in `buildGoModules` and since this only affects Darwin it is fine.